golden_response/staging/get_triples/place_type.json

```diff
@@ -1,5 +1,1951 @@
 {
   "Country": [
+    {
+      "subjectId": "wikidataId/Q958291",
+      "subjectName": "United Principalities",
+      "subjectTypes": [
+        "Country"
+      ],
+      "predicate": "typeOf",
+      "objectId": "Country",
+      "objectName": "Country",
+      "objectTypes": [
+        "Class"
+      ],
+      "provenanceId": "dc/5j06ly1"
+    },
+    {
+      "subjectId": "wikidataId/Q940741",
+      "subjectName": "Republic of Hungary",
+      "subjectTypes": [
+        "Country"
+      ],
+      "predicate": "typeOf",
+      "objectId": "Country",
+      "objectName": "Country",
+      "objectTypes": [
+        "Class"
+      ],
+      "provenanceId": "dc/5j06ly1"
+    },
+    {
+      "subjectId": "wikidataId/Q93180",
+      "subjectName": "Seleucid Empire",
+      "subjectTypes": [
+        "Country"
+      ],
+      "predicate": "typeOf",
+      "objectId": "Country",
+      "objectName": "Country",
+      "objectTypes": [
+        "Class"
+      ],
+      "provenanceId": "dc/5n63hr1"
+    },
+    {
+      "subjectId": "wikidataId/Q903779",
+      "subjectName": "United Kingdom of Portugal, Brazil and the Algarves",
+      "subjectTypes": [
+        "Country"
+      ],
+      "predicate": "typeOf",
+      "objectId": "Country",
+      "objectName": "Country",
+      "objectTypes": [
+        "Class"
+      ],
+      "provenanceId": "dc/5j06ly1"
+    },
+    {
+      "subjectId": "wikidataId/Q8733",
+      "subjectName": "Qing dynasty",
```

Comment:
Is ancient country expected?

Reply:
Nope, it is not good. All these entities are indirect subclasses of country.

I'll file a bug to drop historical stuff, that will need more pruning.

Reply:
But I think we should go ahead with this anyway so we can test it out and find more bugs like this.

Reply:
can you file an issue with the the golden files to revert back later.

Reply:
https://github.com/datacommonsorg/mixer/issues/297